pkg/schema/external.go

```diff
@@ -57,8 +59,6 @@ func (external) JSONSchema() *jsonschema.Schema {
 
 	return &jsonschema.Schema{
 		Type:                 &jsonschema.Type{Types: []string{"object"}},
-		Title:                "external",
-		Description:          "A list of external resources that are part of your development environment",
```

Comment:
Just curiosity, why removing the title and description of this item?

Reply:
The title for root-level properties are defined in the struct with the `jsonschema` annotation, these were getting ignored